### ISSUE TITLE
Tests fails with Hash-Merge 2.999: Can't call method "resultset" on an undefined value at /usr/share/perl5/vendor_perl/DBIx/Class/Schema.pm line 547.

### ISSUE THREAD

**Issue by ppisar** [ironcamel/Dancer2-Plugin-DBIC#24]
If DBD::SQLite is installed, Dancer2-Plugin-DBIC executes optional tests in t/03-dynamic-schemas.t. These passed until upgrading Hash-Merge from 0.200 to 0.299:

```
t/02-multiple-schemas.t .. ok
Can't call method "resultset" on an undefined value at /usr/share/perl5/vendor_perl/DBIx/Class/Schema.pm line 547.
# Looks like your test exited with 255 just after 1.
t/03-dynamic-schemas.t ... 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 2/3 subtests 
```
Also the t/04-testapp.t fails.

**Comment by ppisar** [comment]
This was fixed in DBIx-Class-Schema-Loader-0.07048.

**Comment by ironcamel** [comment]
That's great.  Thanks for the update.
